src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/LevelScattering/src/ACEify.hpp

```diff
@@ -0,0 +1,4 @@
+void ACEify( Table::Data& tData, int ){
+  tData.XSS().push_back( QValue );
+  tData.XSS().push_back( atomicWeightRatio );
```

Comment:
I think this is wrong: LDAT(1) = (A+1)/A*abs(Q) and LDAT(2) = ( A / ( A + 1 ) )^2, unless your qvalue and atomicWeightRatio actually mean these values, in which case you shoud change their name or provide comments to what is actually expected.

Reply:
By the way: section 4.3.11.3 has no table with the content, as seen in the other sections.

Reply:
Done and Done.